deploy/deploy_research_environment.yml

```diff
@@ -5,6 +5,12 @@
   roles:
     - {role: iptables, when: "iptables_config"}
     - {role: common, when: "install_glusterfs"}
+  tasks:
+    - name: research eod symlinks
+      file:
+         src: []
+         dest: []
+         state: link 
```

Comment:
@olivermel This isn't what I meant, rather use `with_items: eod_symlinks`, and `src: "{{ item.src }}"`, `dest: "{{ item.dest }}`.  Then, in `group_vars/all/main.yml`, add a default setting which is `eod_symlinks: []`.  That way, by default no symlinks are created, but that variable can be overridden like so:
```
eod_symlinks:
  - src: /some/path
    dest: /some/other/path
```